manifests/base/webhook/manifests.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: system
```

Comment:
We have no way to modify this namespace since this manifest is automatically generated by the controller-gen. But, this namespace is replaced by https://github.com/kubeflow/training-operator/blob/a618d1c3f77f769da7d4f535e25db88d64aadb88/manifests/base/webhook/kustomizeconfig.yaml.